scipy/stats/unuran/src/COPYING

```diff
@@ -0,0 +1,341 @@
+		    GNU GENERAL PUBLIC LICENSE
```

Comment:
I believe we'll want to remove this GNU license and replace it with the author permission to release under scipy-compatible license

Reply:
I have removed the GNU license. Where can I find the scipy compatible license?